gtk2_ardour/video_timeline.cc

```diff
@@ -33,6 +33,10 @@
 #include "video_timeline.h"
 #include "video_tool_paths.h"
 
+#ifdef PLATFORM_BSD
+#include <sys/wait.h>
+#endif
+
```

Comment:
Why is this needed here?  `WNOHANG` ?  if so it might be better in `libs/pbd/pbd/system_exec.h`
where the `wait()` wrapper is defined.